### PR TITLE
Style tweaks

### PR DESCRIPTION
**Pull request by theophoric**
@amgando check out commit log for explanation of changes.

### CODE REVIEW

File: src/lottery/assembly/index.ts [hunk 1]
```diff
@@ -7,48 +7,48 @@ type AccountId = string;
 @nearBindgen
 export class Contract {
 
-  private _owner: AccountId
```
Comment (theophoric) [inline src/lottery/assembly/index.ts:10]: `_` for private variables isn't really used -- exception is when overriding get / set methods for private class property
e.g.
```ts
private _var: string;
get var(): string {
  return this._var;
}
```

File: src/lottery/assembly/index.ts [hunk 2]
```diff
@@ -7,48 +7,48 @@ type AccountId = string;
...
   constructor(owner: AccountId) {
-    this._owner = owner;
+    this.owner = owner;
   };
 
   // --------------------------------------------------------------------------
   // Public VIEW methods
   // --------------------------------------------------------------------------
 
-  owner(): AccountId {
```
Comment (theophoric) [inline src/lottery/assembly/index.ts:26]: @nearBindgen does not allow for property `get` methods; perhaps it should generate getters for public properties ?

to be consistent with rust interface conventions, changing these methods to `get_XXX()`

File: src/lottery/assembly/index.ts [hunk 3]
```diff
@@ -82,72 +82,77 @@ export class Contract {
...
+      this.payout();
     } else {
-      this.loser()
+      this.loser();
     }
   }
 
   @mutateState()
   setFeeStrategy(strategy: StrategyType): bool {
-    assert(Context.predecessor == Context.contractName, "Only this contract may call itself")
```
Comment (theophoric) [inline src/lottery/assembly/index.ts:94]: extract common assertions to private helper methods

File: src/lottery/assembly/fee-strategies.ts [hunk 4]
```diff
@@ -1,36 +1,36 @@
 import { u128 } from "near-sdk-as";
 
 export enum StrategyType {
-  constant = 0 as u8,
```
Comment (theophoric) [inline src/lottery/assembly/fee-strategies.ts:4]: capitalize enum names and use standard values unless explicitly needed (e.g. values are defined outside of code/context and needs to be explicitly fixed to it)

**Comment by amgando** [comment]
thanks @theophoric!  much appreciated

**Pull request merged**